lib/active_merchant/billing/gateways/conekta.rb

```diff
@@ -82,9 +82,9 @@ def add_order(post, money, options)
 
       def add_details_data(post, options)
         details = {}
-        details[:name] = options[:customer] if options[:customer]
+        details[:name] = options[:customer] || options[:billing_address][:name]
         details[:email] = options[:email] if options[:email]
-        details[:phone] = options[:phone] if options[:phone]
+        details[:phone] = options[:phone] || options[:billing_address][:phone] || options[:shipping_address][:phone]
```

Comment:
Let's actually not look in the shipping address for this, since it could be different from what it should be, even if it isn't provided, in which case it does need to be added to the options/billing address.

Reply:
@curiousepic just pushed updated that only checks billing_address.